dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataOwnerApi.kt

```diff
@@ -0,0 +1,129 @@
+package org.dataland.datalandbackend.api
+
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/ecf391af-0959-4b41-ab30-ac7fbfe7b096)
 If i try to delete a data owner for a company which does not have an owener, i get the error seen above. This is a bit misleading. The company does exist, but has no owners. The error should be clarified.

Reply:
ok, closing this